tests/ccl_test_emu_nu.c

```diff
@@ -122,12 +122,12 @@ static void compare_emu_nu(int i_model,struct emu_nu_data * data)
   char fname[256],str[1024];
   char* rtn;
   FILE *f;
-  int i_model_vec[4]={38,39,40,42}; //The emulator cosmologies we can compare to 
-  
+  int i_model_vec[4]={38,39,40,42}; //The emulator cosmologies we can compare to
+
   ccl_configuration config = default_config;
-  config.transfer_function_method = ccl_emulator;
+  config.transfer_function_method = ccl_boltzmann_class;
```

Comment:
With the new scheme, what's the impact of using `ccl_boltzmann_class` here vs. using `ccl_bbks` in `ccl_test_emu.c`?

Reply:
Speed.